src/main/java/bio/terra/service/filedata/google/gcs/GcsIO.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.service.filedata.google.gcs;
+
+import bio.terra.service.resourcemanagement.exception.GoogleResourceException;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobInfo;
+import com.google.cloud.storage.Storage;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+
+public class GcsIO {
```

Comment:
Maybe a more general name would be `GcsUtils` and we could move the code that converts bucket/path or blobids to GS paths here too?